### PR TITLE
Add section detailling testing with random instance termination

### PR DESCRIPTION
**Pull request by SgtCoDFish**
Includes Netflix "Chaos Monkey" and BBC "chaos-lambda".

These tools are quite popular with businesses running AWS at scale and are worth mentioning as they help engineers to create resilient architectures on AWS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -501,6 +501,7 @@ This guide is about AWS, not DevOps or server configuration management in genera
...
+-   **Testing immutable infrastructure:** If you want to be proactive about testing your service's ability to cope with instance termination or failure, it can be helpful to introduce random instance termination during business hours which will expose any such issues at a time when engineers are available to identify and fix them. Netflix's [Simian Army](https://github.com/Netflix/SimianArmy) (specifically, [Chaos Monkey](https://github.com/Netflix/SimianArmy/wiki/Chaos-Monkey)) is a popular tool for this. Alternatively, [chaos-lambda](https://github.com/bbc/chaos-lambda) by the BBC is a lightweight option which runs on AWS [Lambda](#lambda).
```
Comment (ThanosBaskous) [inline README.md]: nit: we use unicode quotes throughout the guide (’ instead of ')

Comment (ThanosBaskous) [inline README.md]: Suggestion: "during business hours, which will"

**Comment by SgtCoDFish** [comment]
👍 Changes made; does that look OK?

**Comment by ThanosBaskous** [comment]
Thank you for your contribution, @SgtCoDFish!

**Pull request merged**